website/docs/cloud-docs/architectural-details/ip-ranges.mdx

```diff
@@ -10,7 +10,7 @@ tfc_only: true
 
 Terraform Cloud uses static IP ranges for certain features such as notifications and VCS connections. These IP ranges are retrievable through the [IP Ranges API](/cloud-docs/api-docs/ip-ranges).
 
--> **Note:** The IP ranges for each feature returned by the IP Ranges API may overlap. Additionally, these published ranges do not currently allow for execution of Terraform runs against local resources.
+-> **Note:** The IP ranges for each feature returned by the IP Ranges API may overlap. Additionally, these published ranges do not currently allow for execution of Terraform runs against local resources, for e.g. CLI-driven runs.
```

Comment:
```suggestion
-> **Note:** The IP ranges for each feature returned by the IP Ranges API may overlap. Additionally, these published ranges do not currently allow for execution of Terraform runs against local resources, such as for CLI-driven runs.
```

Suggest changing "for e.g." to "such as for" to avoid Latin terms and abbreviations